operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -138,36 +138,25 @@ spec:
 #  external_services:
 #
 # **Grafana-specific settings:
-# api_key: API key to access Grafana. API key only requires viewer permissions.
+# api_key: API key to access Grafana, for token-based authentication. API key only requires viewer permissions.
 # display_link: When true, a link to Grafana will be displayed for more dashboards.
-# in_cluster: Set true to indicate that Grafana service is accessible in the Kubernetes cluster.
-# password: Password to be used when making requests to Grafana. User only requires viewer permissions.
-# service: The Kubernetes service name for Grafana. Kiali uses this to connect within the cluster to Grafana.
-#          If "service" and "service_namespace" are empty then "url" is used for communication.
-# service_dashboard_pattern: Search pattern for Grafana Service dashboard.
-# service_namespace: The Kubernetes namespace that holds the Grafana service (if empty, assumes the same value as deployment.namespace)
+# in_cluster: Set true to indicate that the Grafana service is accessible in the Kubernetes cluster.
+# password: Password to be used when making requests to Grafana, for basic authentication. User only requires viewer permissions.
+# service: The Kubernetes service name for Grafana. Only used when in_cluster is true.
+# namespace: The Kubernetes namespace that holds the Grafana service. Only used when in_cluster is true. If empty, assumes the same value as deployment.namespace.
```

Comment:
trivial: all of these settings in this file are ordered alphabetically - can you move this "namespace" up a few lines - after " in_cluster" but before "password".